settings/cronjob.ini.append.php

```diff
@@ -0,0 +1,12 @@
+<?php /*
+
+[CronjobSettings]
+ExtensionDirectories[]=ezplatformsearch
+
+[CronjobPart-frequent]
```

Comment:
eZ Find adds it to frequent, so I figured I would as well since it serves the same purpose. If its not added to frequent the README should probably be updated to make sure users run the cron.